src/next-transpile-modules.js

```diff
@@ -120,12 +120,15 @@ const withTmInitializer = (modules = [], options = {}) => {
 
     // Generate Webpack condition for the passed modules
     // https://webpack.js.org/configuration/module/#ruleinclude
-    const match = (path) =>
-      modulesPaths.some((modulePath) => {
-        const transpiled = path.includes(modulePath);
+    const match = (path) => {
+      const lastEntry = path.split(`${path.sep}node_modules${path.sep}`).slice(-1)[0];
+
+      return modules.some((modulePath) => {
```

Comment:
Just a quick question, somewhat related with what i've commented here:
https://github.com/martpie/next-transpile-modules/issues/151#issuecomment-771957321

Shouldn't `modules.some`be `modulesPaths.some`? Was it intentional to compare the string with the package name instead of the resolved module path?